app/server/static/components/label.vue

```diff
@@ -52,23 +52,17 @@
                 label.label Color
                 div.field.has-addons
                   div.control
-                    a.button(
-                      v-bind:style="{ \
-                        color: newLabel.text_color, \
-                        backgroundColor: newLabel.background_color \
-                      }"
+                    div.form__field
+                      div.form__input
+                        swatches(v-model="newLabel.background_color", colors="basic", \
+                          show-fallback=true,popover-to="", :trigger-style="{ width: '36px', height: '36px' }")
```

Comment:
Instead of having to use the line-continuation `\` you can also set each attribute on its own line (many other place in the codebase currently do this):

```pug
swatches(
  v-model="newLabel.background_color"
  colors="basic"
  backgroundColor: newLabel.background_color
  show-fallback=true,
  popover-to="", 
  v-bind:trigger-style="{ width: '36px', height: '36px' }"
)
```

Reply:
Agreed! Makes it readable.